code/modules/jobs/job_types/bos.dm

```diff
@@ -600,10 +599,16 @@ Senior Scribe
 	H.mind.teach_crafting_recipe(/datum/crafting_recipe/wattz2k)
 	H.mind.teach_crafting_recipe(/datum/crafting_recipe/AER9)
 	H.mind.teach_crafting_recipe(/datum/crafting_recipe/AEP7)
+	H.mind.teach_crafting_recipe(/datum/crafting_recipe/wattz1k)
+	H.mind.teach_crafting_recipe(/datum/crafting_recipe/magnetowattz)
 	H.mind.teach_crafting_recipe(/datum/crafting_recipe/booster)
 	ADD_TRAIT(H, TRAIT_CHEMWHIZ, src)
 	ADD_TRAIT(H, TRAIT_SURGERY_HIGH, src)
 	ADD_TRAIT(H, TRAIT_CYBERNETICIST, src)
+	ADD_TRAIT(H, TRAIT_GUNSMITH_ONE, src)
+	ADD_TRAIT(H, TRAIT_GUNSMITH_TWO, src)
+	ADD_TRAIT(H, TRAIT_GUNSMITH_THREE, src)
+	ADD_TRAIT(H, TRAIT_GUNSMITH_FOUR, src)
```

Comment:
for(var/datum/crafting_recipe/recipe as() in GLOB.knight_gunsmith_recipes)
		H.mind.teach_crafting_recipe(recipe)

(Thats how they get the recipes via one to four)